Mesh_3/examples/Mesh_3/CMakeLists.txt

```diff
@@ -22,172 +22,175 @@ endif()
 # Use Eigen
 find_package(Eigen3 3.1.0 REQUIRED) #(3.1.0 or greater)
 include(CGAL_Eigen3_support)
-
-find_package(VTK QUIET COMPONENTS vtkImagingGeneral vtkIOImage NO_MODULE)
-if(VTK_FOUND)
-  if(VTK_USE_FILE)
-    include(${VTK_USE_FILE})
-  endif()
-  if("${VTK_VERSION_MAJOR}" GREATER "5" OR VTK_VERSION VERSION_GREATER 5)
-    message(STATUS "VTK found")
-    if(TARGET VTK::IOImage)
-      set(VTK_LIBRARIES VTK::ImagingGeneral VTK::IOImage)
+if(TARGET CGAL::Eigen3_support)
```

Comment:
I disagree with the patch. The line `find_package(Eigen3 3.1.0 REQUIRED)` has `REQUIRED`. We do not want to add an extra `if(TARGET CGAL::Eigen3_support)` and then indent the whole file.

Reply:
good catch, I forgot to remove the requiered